src/qs8-vadd/rvv.c.in

```diff
@@ -0,0 +1,71 @@
+// Copyright 2023 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+#include <riscv_vector.h>
+#include <xnnpack/vbinary.h>
+$assert DATATYPE in ["QS8", "QU8"]
+$XINT8_T = {"QS8": "int8_t", "QU8": "uint8_t"}[DATATYPE]
+$VSIZE1 = {1: "m1", 2: "m2"}[LMUL]
+$VSIZE2 = {1: "m2", 2: "m4"}[LMUL]
+$VSIZE4 = {1: "m4", 2: "m8"}[LMUL]
+
```

Comment:
nit: 2 blank lines before the microkernel function

Reply:
Done